src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    if (inverse[robot[key]]) {
+      return null;
+    } else {
+      inverse[robot[key]] = key;
+    }
```

Comment:
```suggestion
    if (inverse[robot[key]]) {
      return null;
    }
    
    inverse[robot[key]] = key;
```
There is no need for `else` as your code will not go further after return anyway